### PR TITLE
Added new tag theme: "broader"

### PR DESCRIPTION
**Pull request by niklabh**
Screenshot from 2022-04-12 13-43-50

Screenshot from 2022-04-12 13-43-34

Screenshot from 2022-04-12 13-42-58

### CODE REVIEW

File: src/components/Tag/Tag.styles.ts [hunk 1]
```diff
@@ -22,6 +22,7 @@ import {
     inactiveStatus,
     initialStyles,
     regular,
+    boarder,
```
Comment (BillyG83) [inline src/components/Tag/Tag.styles.ts:25]: please add props in alphabetical order, there is a plugin you can use. Check the contribute file please =]

Comment (BillyG83) [reply src/components/Tag/Tag.styles.ts:25]: also i got some feedback from design, can you change 'boarder' to 'chips' please

File: src/components/Tag/Tag.styles.ts [hunk 2]
```diff
@@ -66,7 +69,7 @@ const getColors = (color?: string, tone?: Tone) => {
 
 export const SpanStyled = styled.div<TStyleProps>`
     cursor: pointer;
-    margin-left: 16px;
+    margin-left: 8px;
```
Comment (BillyG83) [inline src/components/Tag/Tag.styles.ts:72]: nice i was thinking this looked too far out

File: src/components/Tag/Tag.tsx [hunk 3]
```diff
@@ -39,7 +39,6 @@ const Tag: React.FC<TagProps> = ({
                         fill={colors.blue}
                         size={16}
                         svg="x"
-                        style={{ marginLeft: '8px' }}
```
Comment (BillyG83) [inline src/components/Tag/Tag.tsx:42]: good tidy up TY

**Review by BillyG83 — CHANGES_REQUESTED** [review]
very minor changes needed here, good job man

File: src/components/Tag/Tag.styles.ts (hunk 1 above)
Comment (niklabh) [reply src/components/Tag/Tag.styles.ts]: done

**Pull request merged**